resources/lib/database/queries.py

```diff
@@ -1,4 +1,5 @@
 
+<<<<<<< HEAD
```

Comment:
This is an artefact from the merge conflict.
Personally I would've used the file from master, and just repeated the changes - This file only had a few variables removed IIRC, which is rather quick to repeat

Reply:
Damn, thought I got them all, shame it can't just do my changes over the top of the current master

Reply:
Oh, you did a merge - have a look into `git rebase` - it essentially takes your changes, and re-applies them on top of the changes from another branch - wouldn't have changed the fact that me touching all the files caused a merge conflict tho.

Reply:
I tried that first, all it said was "noop"